tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -29,16 +29,6 @@ build --copt=-mavx --host_copt=-mavx
 build --profile=/tf/pkg/profile.json.gz
 
 # CUDA: Set up compilation CUDA version and paths
-<<<<<<< HEAD
-build:cuda --@local_config_cuda//:enable_cuda
```

Comment:
We need these `:cuda` specifiers here since we have `:rocm` now too. 
The only change here should be the crosstool line